docker/Release.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.mirror.hashicorp.services/alpine:3.10
+FROM alpine:3.13
```

Comment:
@mdeggies I think we need to stick to the hashicorp mirror for this image, is that correct? Do we have one for 3.13?

Reply:
Yes, this should be `docker.mirror.hashicorp.services/alpine:3.13`

Reply:
Cool. Updated now!

Reply:
Just a note, in the future you can run `docker pull docker.mirror.hashicorp.servies/$whatever` locally to see if that image is supported in the docker mirror. If it's not, you can create a PR to add that here: https://github.com/hashicorp/docker-registry-mirror/blob/main/images.tf.